src/components/styles/_stack.less

```diff
@@ -0,0 +1,133 @@
+@import "../../_variables.less";
+@height: 50px;
+@width: @height;
+@rotate: -30deg;
+@skew: 25deg;
+@bgColor: red;
+
+.layer(@height, @width, @rotate, @skew, @bgColor) {
+  content: "";
+  height: @height;
+  width: @width;
+  border-radius: inherit;
+  background: @bgColor;
+  position: absolute;
+  transform: rotate(@rotate) skew(@skew) translate(-10px, 10px);
+}
+
+.design_logic(@bgColor, @height, @width) {
+  // THE DESIGN LOGIC START
+  &:before {
+    .layer(@height, @width, @rotate, @skew, @bgColor);
+    content: "Button";
```

Comment:
Why is this in before? Can't we make it in the text inside the button element instead? Putting it in `:before` makes it hard to customize (as the user will need to use whatever text they have). Please look into it and make necessary adjustments

Reply:
I can't do that as it is important for the design that the text stay inside the before element. I'll see if I actually can spin up some JS to divert user input to the content in the before element. If you have any ideas, please do share !👍🏻

Reply:
Leave it as it is for now and we can open an issue later with it. Maybe someone else knows how to help